Trains News Wire.js

```diff
@@ -0,0 +1,130 @@
+{
+	"translatorID": "3f4f0a1c-f542-4b44-8824-40ee1f921373",
+	"label": "Trains News Wire",
+	"creator": "Charles Fulton",
+	"target": "https?://trn.trains.com/news/news-wire/[0-9]+",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-05-14 13:33:45"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2016 Charles Fulton
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	return "blogPost";
+}
+
+function doWeb(doc, url) {
+	scrape(doc, url);
+}
+
+function scrape(doc, url) {
+	newItem = new Zotero.Item("blogPost");
+	newItem.url = doc.location.href;
```

Comment:
Why not simply use the second parameter `url`?
(In some environments `doc.location` might be problematic...)